e2e/playwright/testing-gizmo.spec.ts

```diff
@@ -1,18 +1,18 @@
-import { test, expect } from '@playwright/test'
-
+import { _test, _expect } from './playwright-deprecated'
```

Comment:
I mustve missed the memo - what's going on here? :)

Reply:
We'll undo this change in a separate PR per Slack discussion